.npmrc

```diff
@@ -1 +1,2 @@
 registry = https://registry.npmjs.org/
+save-exact=true
```

Comment:
I personally think pinning dependencies and explicitly update them with renovate or similar tools is nicer